index.html

```diff
@@ -2,69 +2,70 @@
 <html>
 <head>
   <meta charset="utf-8">
-  <title>My Basic Site
+  <title>Kae Anderson</title>
 
-  Leave these links here...
-  But this should be in a comment...
+  <!-- Leave these links here...
+  But this should be in a comment... -->
   <link href="https://fonts.googleapis.com/css?family=Cormorant|Poppins:300,400" rel="stylesheet">
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="styles/style.css">
 </head>
 <body>
-  <div class="bender">
-    <nav class="bender__nav">
-      <div class="box">
-        <div class="abc">
-          <h1 class="abc__first">Shia LaBeouf</h1>
+  <div class="app">
+    <nav class="app__nav">
+      <div class="app__content">
```

Comment:
So here the original class name was `box` so only the things that said `bender` before would be replaced with `app`.

Here this div is grouping things together to set a max width and bring things together: see if there is a class name that matches that description that doesn't have `__`.